### ISSUE TITLE
待阅读文章列表

### ISSUE THREAD

**Issue by ccc013** [ccc013/Study-Notes#9]
### 机器学习

- [ ] [A Gentle Introduction to Object Recognition With Deep Learning](https://machinelearningmastery.com/object-recognition-with-deep-learning/)--目标检测入门介绍
- [ ] [Cloth Swapping with Deep Learning: Implement Conditional Analogy GAN in Keras](https://shaoanlu.wordpress.com/2017/10/26/reimplement-conditional-anology-gan-in-keras/)
- [ ] [Feature Selection with sklearn and Pandas](https://towardsdatascience.com/feature-selection-with-pandas-e3690ad8504b)
- [ ] [机器学习竞赛小众特征工程技巧](https://medium.com/ml-byte/rare-feature-engineering-techniques-for-machine-learning-competitions-de36c7bb418f)
- [x] [machine-learning-in-python](https://www.pyimagesearch.com/2019/01/14/machine-learning-in-python/)--翻译完成
- [x] [9 Applications of Deep Learning for Computer Vision](https://machinelearningmastery.com/applications-of-deep-learning-for-computer-vision/)--计算机视觉的应用综述，参考+查找资料，写成文章
- [x] [10 New Things I Learnt from fast.ai v3](https://towardsdatascience.com/10-new-things-i-learnt-from-fast-ai-v3-4d79c1f07e33)：阅读，主要是对 fast.ai 课程学习后的感想

#### PyTorch

- [ ] [A Simple Neural Network from Scratch with PyTorch and Google Colab](https://medium.com/dair-ai/a-simple-neural-network-from-scratch-with-pytorch-and-google-colab-c7f3830618e0)
- [ ] [Pytorch--采用CNN的迁移学习](https://towardsdatascience.com/transfer-learning-with-convolutional-neural-networks-in-pytorch-dd09190245ce)
- [ ] [PyTorch实现的BiSeNet人脸解析改进](https://github.com/zllrunning/face-parsing.PyTorch)
- [ ] [Pytorch可视化工具--tensorwatch](https://github.com/microsoft/tensorwatch)
- [x] [加速PyTorch](https://towardsdatascience.com/speed-up-your-algorithms-part-1-pytorch-56d8a4ae7051)
- [x] [PyTorch数据加载教程](https://pytorch.org/tutorials/beginner/data_loading_tutorial.html)

#### Keras

- [ ] [Keras Tutorial: How to get started with Keras, Deep Learning, and Python](https://www.pyimagesearch.com/2018/09/10/keras-tutorial-how-to-get-started-with-keras-deep-learning-and-python/)
- [ ] [Fine-tuning with Keras and Deep Learning](https://www.pyimagesearch.com/2019/06/03/fine-tuning-with-keras-and-deep-learning/)
- [ ] [Faster R-CNN (object detection) implemented by Keras for custom data from Google’s Open Images Dataset V4](https://towardsdatascience.com/faster-r-cnn-object-detection-implemented-by-keras-for-custom-data-from-googles-open-images-125f62b9141a)--Keras 训练 Faster R-CNN
- [ ] [(Keras)用迁移学习从人脸图像估计体质指数](https://medium.com/@leosimmons/estimating-body-mass-index-from-face-images-using-keras-and-transfer-learning-de25e1bc0212)
- [ ] [可视化卷积核和 features map](https://machinelearningmastery.com/how-to-visualize-filters-and-feature-maps-in-convolutional-neural-networks/)

**Comment by ccc013** [comment]
### 深度学习

- [ ] [谷歌Colab的免费GPU使用教程](https://medium.com/deep-learning-turkey/google-colab-free-gpu-tutorial-e113627b9f5d)

#### CNN

- [ ] [Simple Introduction to Convolutional Neural Networks](https://towardsdatascience.com/simple-introduction-to-convolutional-neural-networks-cdf8d3077bac)
- [ ] [Advanced Topics in Deep Convolutional Neural Networks](https://towardsdatascience.com/advanced-topics-in-deep-convolutional-neural-networks-71ef1190522d)
- [ ] [Intuitively Understanding Convolutions for Deep Learning](https://towardsdatascience.com/intuitively-understanding-convolutions-for-deep-learning-1f6f42faee1)
- [ ] [How to Visualize Filters and Feature Maps in Convolutional Neural Networks](https://machinelearningmastery.com/how-to-visualize-filters-and-feature-maps-in-convolutional-neural-networks/)--可视化卷积核和特征图
- [ ] [How to Implement VGG, Inception and ResNet Modules for Convolutional Neural Networks from Scratch](https://machinelearningmastery.com/how-to-implement-major-architecture-innovations-for-convolutional-neural-networks/)--实现VGG、Inception、ResNet模块代码
- [ ] [A Simple Guide to the Versions of the Inception Network](https://towardsdatascience.com/a-simple-guide-to-the-versions-of-the-inception-network-7fc52b863202)
- [ ] [CNN Heat Maps: Class Activation Mapping (CAM)](https://towardsdatascience.com/cnn-heat-maps-class-activation-mapping-cam-614180e360d5?MGa=true)
- [ ] [【俗话卷积网络】《Convolutional Neural Networks — Simplified》](https://medium.com/x8-the-ai-community/cnn-9c5e63703c3f)
- [ ] [如何区分并记住常见的几种 Normalization 算法](https://mp.weixin.qq.com/s/zPEdUDetu15zmitQeGz9WA)
- [ ] [【完结】总结12大CNN主流模型架构设计思想](https://mp.weixin.qq.com/s/VDaBk_uNfrKaWYkPbtC3xQ)
- [ ] [CNN网络结构的发展：从LeNet到EfficientNet](https://mp.weixin.qq.com/s/ooK2aAC_TAPFmK9dPLF-Fw)
- [ ] [如何训练ResNet系列文章](https://myrtle.ai/learn/)--一共八篇文章


### 计算机视觉

- [ ] [利用复联3介绍 Image Caption](https://medium.com/analytics-vidhya/introduction-to-image-caption-generation-using-the-avengers-infinity-war-characters-6f14df09dbe5)
- [ ] [计算摄影 | 图像合成与融合](https://mp.weixin.qq.com/s/QzlaX4HtYmR5EAgi5_L16g)
- [ ] [mAP (mean Average Precision) for Object Detection](https://medium.com/@jonathan_hui/map-mean-average-precision-for-object-detection-45c121a31173)
- [ ] [图像处理100问](https://github.com/gzr2017/ImageProcessing100Wen)--Github 上的一个图像处理教程
- [ ] [深度学习超分辨率指南](https://medium.com/beyondminds/an-introduction-to-super-resolution-using-deep-learning-f60aff9a499d)
- [x]  [【技术综述】多标签图像分类综述](https://mp.weixin.qq.com/s/6K4tXPlYLaXhexh6gElP5Q)

#### 项目

- [ ] [The 4 Convolutional Neural Network Models That Can Classify Your Fashion Images](https://towardsdatascience.com/the-4-convolutional-neural-network-models-that-can-classify-your-fashion-images-9fe7f3e5399d)--介绍了4种CNN模型来训练时尚数据集
- [ ] [手把手教你搭建食物识别API](https://reshamas.github.io/deploying-deep-learning-models-on-web-and-mobile/)
- [x] [利用CycleGAN生成不同属性的神奇宝贝](https://mp.weixin.qq.com/s/0xn57qy2CQiUvbF_wWwrlw)
- [ ] [How to implement a YOLO (v3) object detector from scratch in PyTorch: Part 1](https://blog.paperspace.com/how-to-implement-a-yolo-object-detector-in-pytorch/)--pytorch实现YOLO(v3)的一系列教程
- [ ] [用深度学习和局部敏感哈希搜索相似图片](https://towardsdatascience.com/finding-similar-images-using-deep-learning-and-locality-sensitive-hashing-9528afee02f5)
- [x] [手把手系列 | 教你用Python构建多标签图像分类模型（附案例）](https://mp.weixin.qq.com/s/xP5v-tKZNyNtZHhOqO-J4w)
- [x] [图像配准：从SIFT到深度学习](https://medium.com/m/global-identity?redirectUrl=https%3A%2F%2Fblog.sicara.com%2Fimage-registration-sift-deep-learning-3c794d794b7a)

### GAN

- [x] [GANs"必读"论文推荐](https://towardsdatascience.com/must-read-papers-on-gans-b665bbae3317)
- [x] [GAN的有趣应用](https://machinelearningmastery.com/impressive-applications-of-generative-adversarial-networks/)

#### 项目

- [ ] [GANs的端到端介绍](https://towardsdatascience.com/an-end-to-end-introduction-to-gans-bf253f1fa52f)


### 人脸

#### 项目

- [ ] [#KnowYourAI: Developing a Framework to Address Bias in Facial Expression Recognition](https://towardsdatascience.com/knowyourai-developing-a-framework-to-address-bias-in-facial-expression-recognition-b3b8040b0a68)
- [ ] [Implementing Face Detection using Python and OpenCV](https://medium.com/analytics-vidhya/how-to-build-a-face-detection-model-in-python-8dc9cecadfe9)--利用 opencv 实现人脸检测
- [ ] [How to Perform Face Detection with Deep Learning in Keras](https://machinelearningmastery.com/how-to-perform-face-detection-with-classical-and-deep-learning-methods-in-python-with-keras/)--基于深度学习方法的人脸检测
- [ ] [一款入门级的人脸、视频、文字检测以及识别的项目](https://github.com/vipstone/faceai)

**Comment by ccc013** [comment]
### Python

- [x] [Jupyter Notebook 入门教程](https://www.dataquest.io/blog/jupyter-notebook-tutorial/)
- [x] [Jupyter Notebook进阶教程](https://www.dataquest.io/blog/advanced-jupyter-notebooks-tutorial/)
- [ ] [强化你的Jupyter Notebooks](https://towardsdatascience.com/supercharging-jupyter-notebooks-e22f5ad7ca18)
- [ ] [Reading and Writing CSV Files in Python](https://realpython.com/python-csv/)--读取，保存csv文件
- [ ] [How to create, read, update and search through Excel files using Python](https://medium.freecodecamp.org/how-to-create-read-update-and-search-through-excel-files-using-python-c70680d811d4)
- [ ] [Python Tricks 101](https://hackernoon.com/python-tricks-101-2836251922e0)--总结了一些不常见当有效的技巧，主要来自 PyTricks 项目
- [ ] [Jupyter tips, tricks and shortcuts](https://www.dataquest.io/blog/jupyter-notebook-tips-tricks-shortcuts/)--28 个 Jupyter 的 秘诀、技巧和快捷键
- [ ] [技巧、诀窍和魔法：轻松优化Jupyter Notebook](https://towardsdatascience.com/how-to-effortlessly-optimize-jupyter-notebooks-e864162a06ee)
- [x] [用Numba加速Python代码](https://towardsdatascience.com/heres-how-you-can-get-some-free-speed-on-your-python-code-with-numba-89fdc8249ef3)
- [ ] [Python 工匠：高效操作文件的三个建议](https://mp.weixin.qq.com/s/iXOqe5I00sVE05qVjY0ZIw)
- [ ] [24式加速你的Python](https://mp.weixin.qq.com/s?__biz=MzU5MDY5OTI5MA==&mid=2247484643&idx=1&sn=55ba87185102042bf8c4641e14573161&chksm=fe3b0a96c94c83802ef2a8c4f35fb2364bf732f7931739a3d09386fa244634720b271892fe5b&token=143645418&lang=zh_CN#rd)--阅读总结
- [ ] [Python 工匠：让函数返回结果的技巧](https://mp.weixin.qq.com/s/kzO6jQuMo8-MJNvXM7lZnw)
- [x] [Python Pro Tip: Start using Python defaultdict and Counter in place of dictionary](https://towardsdatascience.com/python-pro-tip-start-using-python-defaultdict-and-counter-in-place-of-dictionary-d1922513f747)
- [x] [Cython 加速Python](https://towardsdatascience.com/use-cython-to-get-more-than-30x-speedup-on-your-python-code-f6cb337919b6)

#### 项目

- [ ] [Python 批量下载无版权图片](https://mp.weixin.qq.com/s/lqbb3Wpok7a9yvshgmaVuw)

### Git

- [ ] [Git/GitHub基础速查](https://medium.com/better-programming/basic-git-github-cheat-sheet-fa020831cb35)

**Comment by ccc013** [comment]
### 完成

- [x] [machine-learning-in-python](https://www.pyimagesearch.com/2019/01/14/machine-learning-in-python/)--翻译完成
- [x] [9 Applications of Deep Learning for Computer Vision](https://machinelearningmastery.com/applications-of-deep-learning-for-computer-vision/)--计算机视觉的应用综述，参考+查找资料，写成文章
#### 0528

- [x] [10 New Things I Learnt from fast.ai v3](https://towardsdatascience.com/10-new-things-i-learnt-from-fast-ai-v3-4d79c1f07e33)：阅读，主要是对 fast.ai 课程学习后的感想

**Comment by ccc013** [comment]
### 项目实战

#### Python


####爬虫

- [ ] [Python 批量下载无版权图片](https://mp.weixin.qq.com/s/lqbb3Wpok7a9yvshgmaVuw)

#### 机器学习/深度学习
